Assets/Scripts/Rune.cs

```diff
@@ -4,6 +4,9 @@
 
 public class Rune : MonoBehaviour
 {
+    // Reference to the script with a queue of all runes
+    public GlobalRunes all_runes;
```

Comment:
This variable should be named `allRunes`. I think for C# conventions, the variables names generally don't have underscores

Reply:
Got it, changed